openhtf/plugs/usb/adb_device.py

```diff
@@ -30,7 +30,7 @@
 class, subclass, and protocol.
 """
 
-import cStringIO
+import io
```

Comment:
Maintainers, maybe someone can comment on whether cStringIO is strictly required on Python2, vs regular io?

Reply:
I don't know of any reason why cStringIO is strictly required, I believe it was potentially used for performance reasons?  i don't believe this adb_device plug is actually used by anyone, ADB interfaces seem to be everyone's favorite thing to implement themselves :)

Reply:
it was definitely for performance compared to StringIO and due to ignorance of the io module